build-docker-image.sh

```diff
@@ -48,9 +48,19 @@ function get_compatible_dockerfile_for_os_or_error
 {
     local version=${1}
     local requested_os=${2}
+
     local major=$(echo "${version}" | sed -E 's/^([0-9]+)\.([0-9]+)\..*/\1/')
     local minor=$(echo "${version}" | sed -E 's/^([0-9]+)\.([0-9]+)\..*/\2/')
+
     case ${major} in
+        # Version is calver
+        2024)
+                local SUPPORTED_IMAGE_OS=("debian" "ubi9" "ubi8")
```

Comment:
could you remove "ubi8" from here, we don't support it any more.
(If you could also remove `"ubi8"` from the `5)` section of the switch/case that would be great too :slightly_smiling_face: )